endor/.gitignore

```diff
@@ -0,0 +1,33 @@
+# Logs
```

Comment:
This file shouldn't be needed. 1 gitignore for the entire repo is enough at the top most directory